test/packages/with-custom-agent/auditd_manager_independent_agent/data_stream/auditd/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -0,0 +1,333 @@
+---
+description: Pipeline for parsing Linux auditd logs
+processors:
+  - set:
+      field: ecs.version
+      value: '8.2.0'
+  - rename:
+      field: event.original
+      target_field: auditd.messages
+      ignore_missing: true
+  - join:
+      field: auditd.messages
+      separator: "\n"
+      target_field: event.original
+      if: ctx?.auditd?.messages != null && ctx.auditd.messages instanceof List
+  - rename:
+      field: error.message
+      target_field: auditd.warnings
+      ignore_missing: true
+  - join:
+      field: auditd.warnings
+      separator: "\n"
+      target_field: error.message
+      if: ctx?.auditd?.warnings != null && ctx.auditd.warnings instanceof List
+  # if it is not a list, rename it back to error.message
+  - set:
+      field: error.message
+      copy_from: auditd.warnings
+      ignore_empty_value: true
+      if: ctx?.auditd?.warnings != null && !(ctx.auditd.warnings instanceof List)
```

Comment:
Added this new step in the pipeline to detect if there is just one warning (string)

![warnings as a string](https://github.com/elastic/elastic-package/assets/5330827/4173aaa4-8920-4ecc-9648-eaa8d56217bf)

Adding this would make fail this test package when ELASTIC_PACKAGE_TEST_ENABLE_INDEPENDENT_AGENT is false, since this will use the Elastic Agent from the stack that does not have the required capabilities.

Should we skip this test package in the CI scripts ? @jsoriano

Reply:
> Should we skip this test package in the CI scripts ? @jsoriano

Yeah, we will need to find a way to test this package only when independent agents are enabled.